frontend/src/components/App/styles.module.css

```diff
@@ -0,0 +1,4 @@
+.root {
+  max-width: 800px;
```

Comment:
Any reason for this number?

Reply:
Nah, it's pretty arbitrary for now and will likely change when we start displaying the board!